optional_requirements.txt

```diff
@@ -8,10 +8,7 @@
 ## of pip from the website, and then possibly install/upgrade setuptools using
```

Comment:
If all dependencies are now in `setup_requirements.py`, I would remove this file (and possibly move comments in there)